Sources/MapboxNavigation/VisualInstruction.swift

```diff
@@ -113,8 +116,10 @@ extension VisualInstruction {
     public func carPlayManeuverLabelAttributedText<T: InstructionLabel>(bounds: @escaping () -> (CGRect),
                                                                         shieldHeight: CGFloat,
                                                                         window: UIWindow?,
+                                                                        customTraitCollection: UITraitCollection? = nil,
```

Comment:
I think this parameter would be just as clear with `traitCollection` as the label, avoiding the vague and overloaded word “custom”.